Reference/Querying/Udi.md

```diff
@@ -26,7 +26,15 @@ Breaking it down:
 
 You can use UDIs in several of the Querying and Management/Service APIs. Read more about the [UDI API reference](https://our.umbraco.com/apidocs/v7/csharp/api/Umbraco.Core.Udi.html).
```

Comment:
Let's add the Umbraco 8 version here as well.
I'm thinking something along these lines:

"You can use UDIs in several of the Querying and Management/Service APIs. Read more about it in our API Documentation for https://our.umbraco.com/apidocs/v7/csharp/api/Umbraco.Core.Udi.html and [Umbraco 8](https://our.umbraco.com/apidocs/v8/csharp/api/Umbraco.Core.Udi.html)."

What do you think? 🤔